io/disk/vfc-tests/virtual_fc.py

```diff
@@ -357,13 +395,24 @@ def vfchost_define_undefine(self, operation, vfc_dic):
         if output.exit_status != 0:
             self.log.debug(output.stderr)
             self.fail("vfchost %s operation failed" % operation)
+        else:
+            self.log.info("%s of %s succes" % (operation, vfc_dic["vfchost"]))
+        time.sleep(5)
```

Comment:
Can the static sleep times be changed into something dynamic based on some outcome ?

Reply:
I will think on it, but due to multiple functionality verification, which takes different time it would be difficult.
my might end up writing more for loops :-)

Reply:
That is fine, if the logic of wait is fool proof. We can scale in future without editing the script at all.

Reply:
@narasimhan-v we will have a call on this, for now we cannot fix. if conclude with some way after our call then  I will send this in another PR. for now we can consider it.